### ISSUE TITLE
Kebesei

### ISSUE THREAD

**Issue by s-o-u-a-n-g** [tekinged/missing#439]
Evening

**Comment by johnbent** [comment]
What is the difference btwn kebesei and klebesei?

**Comment by s-o-u-a-n-g** [comment]
Kebesei is another way to say evening "kebesengei". 

Klebesei is night time or day, as in some day or one day.

**Comment by Gnalu** [comment]
there's 'oingei'-oingerang
            'eliseb'-idelisb
            'kukuk'-klukuk
            'cheos'-chedecheos
            'aita sils'-ieta sils

**Comment by s-o-u-a-n-g** [comment]
> there's 'oingei'-oingerang
> 
>             'eliseb'-idelisb
> 
>             'kukuk'-klukuk
> 
>             'cheos'-chedecheos
> 
>             'aita sils'-ieta sils
> 
>             
> 
> 

Yes I love all those because my grand and great grand parents talked like that and even my aunts and uncles, still. It affects how we speak so even my young cousins use these words. I like it.
